packages/core/src/types/IViewport.ts

```diff
@@ -47,6 +47,10 @@ interface IViewport {
   getActors(): Array<ActorEntry>;
   /** returns specific actor by its uid */
   getActor(actorUID: string): ActorEntry;
+  /** returns specific actor uid by array index */
+  getActorUIDbyIndex(index: number): string;
+  /** returns specific actor by array index */
+  getActorbyIndex(index: number): ActorEntry;
```

Comment:
Fix the camelCase to getActorByIndex

Reply:
done https://github.com/cornerstonejs/cornerstone3D-beta/pull/110/commits/051517ca9dc6b1b312e0aba6a466a801a9e3d1d5